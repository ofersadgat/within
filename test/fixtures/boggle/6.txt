btwsa
dhrdy
inylc
eiano
eyeti