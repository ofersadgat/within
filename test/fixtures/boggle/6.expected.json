{"word":"dryland","score":26}